eve_sqlalchemy/tests/config/resourceconfig/schema.py

```diff
@@ -66,6 +69,10 @@ def test_required(self):
         # autoincrement='auto' per default (see SQLAlchemy docs for
         # details). As such, it is not required.
         self.assertFalse(schema['id']['required'])
+        # Columns with server_default field should be marked as not required
+        # additional test for nullable field set to True
+        self.assertFalse(schema['a_nullable_boolean']['required'])
+        self.assertFalse(schema['a_server_default_col']['required'])
 
```

Comment:
The comment does not provide additional information here, you can safely remove it.